<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>926</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <property name="styleSheet">
         <string notr="true">border-bottom: 1px solid black;</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Temperature:</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLCDNumber" name="temperatureValue">
             <property name="smallDecimalPoint">
              <bool>true</bool>
             </property>
             <property name="digitCount">
              <number>4</number>
             </property>
             <property name="segmentStyle">
              <enum>QLCDNumber::Flat</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_8">
             <property name="styleSheet">
              <string notr="true">font-size: 14px;</string>
             </property>
             <property name="text">
              <string>Â°C</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="styleSheet">
         <string notr="true">border-bottom: 1px solid black;</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Humidity:</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLCDNumber" name="humidityValue">
             <property name="smallDecimalPoint">
              <bool>true</bool>
             </property>
             <property name="digitCount">
              <number>4</number>
             </property>
             <property name="segmentStyle">
              <enum>QLCDNumber::Flat</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="styleSheet">
              <string notr="true">font-size: 14px;</string>
             </property>
             <property name="text">
              <string>%</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <property name="styleSheet">
         <string notr="true">border-bottom: 1px solid black;</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Moisture:</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLCDNumber" name="moistureValue">
             <property name="smallDecimalPoint">
              <bool>true</bool>
             </property>
             <property name="digitCount">
              <number>4</number>
             </property>
             <property name="segmentStyle">
              <enum>QLCDNumber::Flat</enum>
             </property>
             <property name="value" stdset="0">
              <double>0.000000000000000</double>
             </property>
             <property name="intValue" stdset="0">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="styleSheet">
              <string notr="true">font-size: 14px;</string>
             </property>
             <property name="text">
              <string>%</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_5" native="true">
        <property name="styleSheet">
         <string notr="true">border-bottom: 1px solid black;</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_11">
         <item>
          <widget class="QLabel" name="modeLabel">
           <property name="styleSheet">
            <string notr="true">font-size: 12px;</string>
           </property>
           <property name="text">
            <string>NULL</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="regimeList"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>300</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QWidget" name="HeatButtonWidget" native="true">
            <property name="styleSheet">
             <string notr="true">QWidget#HeatButtonWidget{border: 1px solid black;}
</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_9">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_8">
               <item>
                <widget class="QLabel" name="label_10">
                 <property name="maximumSize">
                  <size>
                   <width>70</width>
                   <height>70</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="pixmap">
                  <pixmap resource="../Resources.qrc">:/icons/assets/icons/Local_Fire_Department_Icon_1.png</pixmap>
                 </property>
                 <property name="scaledContents">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="heatLabel">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QSlider" name="heatSlider">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimum">
                <number>150</number>
               </property>
               <property name="maximum">
                <number>400</number>
               </property>
               <property name="singleStep">
                <number>5</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="RainButtonWidget" native="true">
            <property name="styleSheet">
             <string notr="true">QWidget#RainButtonWidget{border: 1px solid black;}</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLabel" name="label_12">
                 <property name="styleSheet">
                  <string notr="true">border: none;</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="pixmap">
                  <pixmap resource="../Resources.qrc">:/icons/assets/icons/Invert_Colors_Icon_1.png</pixmap>
                 </property>
                 <property name="scaledContents">
                  <bool>false</bool>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                 <property name="wordWrap">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="moistureLabel">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QSlider" name="moistureSlider">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximum">
                <number>100</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="LightButtonWidget" native="true">
            <property name="styleSheet">
             <string notr="true">QWidget#LightButtonWidget{border: 1px solid black;}
</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="maximumSize">
                  <size>
                   <width>60</width>
                   <height>60</height>
                  </size>
                 </property>
                 <property name="layoutDirection">
                  <enum>Qt::LeftToRight</enum>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="pixmap">
                  <pixmap resource="../Resources.qrc">:/icons/assets/icons/Lightbulb_Outline_Icon_1.png</pixmap>
                 </property>
                 <property name="scaledContents">
                  <bool>true</bool>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="lightLabel">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QSlider" name="lightSlider">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximum">
                <number>100</number>
               </property>
               <property name="singleStep">
                <number>10</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="WindButtonWidget" native="true">
            <property name="styleSheet">
             <string notr="true">QWidget#WindButtonWidget{border: 1px solid black;}
</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_10">
               <item>
                <widget class="QLabel" name="label_13">
                 <property name="styleSheet">
                  <string notr="true">border: none;</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="pixmap">
                  <pixmap resource="../Resources.qrc">:/icons/assets/icons/Waves_Icon_1.png</pixmap>
                 </property>
                 <property name="scaledContents">
                  <bool>false</bool>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="windLabel">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QSlider" name="windSlider">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximum">
                <number>100</number>
               </property>
               <property name="singleStep">
                <number>10</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="saveButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>700</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>926</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHome">
    <property name="title">
     <string>Home</string>
    </property>
    <addaction name="actionEdit_modes"/>
   </widget>
   <addaction name="menuHome"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionEdit_modes">
   <property name="text">
    <string>Edit modes</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../Resources.qrc"/>
 </resources>
 <connections/>
</ui>
